csharp/src/Microsoft.ML.OnnxRuntime/NativeMethods.shared.cs

```diff
@@ -1824,4 +1826,105 @@ internal class NativeLib
 
 #endregion
     } //class NativeMethods
+
+    internal static class EmptyArray<T>
```

Comment:
>internal static class EmptyArray<T> [](http://example.com/codeflow?start=4&length=35)

Can this be moved to utilities?

Reply:
Certainly. I'm not overly familiar with the codebase so wasn't sure where the best place to put various things would be.